egs2/fisher_callhome_spanish/asr1/local/data.sh

```diff
@@ -69,7 +64,17 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
 
     # concatenate multiple utterances
     local/normalize_trans.sh ${sfisher_transcripts} ${callhome_transcripts}
-    ./utils/combine_data.sh data/dev_all data/dev data/dev2
+
+    utils/combine_data.sh \
+        --extra-files "text.lc.en text.lc.es text.lc.rm.en text.lc.rm.es text.tc.en text.tc.es" \
+        data/train \
+        data/fisher_train data/callhome_train/ 
+
+    cp -r data/fisher_dev data/dev
```

Comment:
Why not use both dev sets for validation here?

Reply:
This is just for consistency with espnet1 st, previous papers seem to only use fisher dev 1 ref for validation.